src/components/Confirm/index.js

```diff
@@ -67,12 +69,12 @@ class CaptureViewer extends Component {
 
 const RetakeAction = ({retakeAction, i18n}) =>
   <button onClick={retakeAction}
-    className={`${theme.btn} ${style["btn-outline"]}`}>
+    className={`${theme.btn} ${style["btn-outline"]}`}>{ /*`*/ }
```

Comment:
This looks like a mistaken commit?

Reply:
It is a mistaken commit.

I added this to be able to use Chrome dev tool's debugger, needed because of this bug:

https://bugs.chromium.org/p/chromium/issues/detail?id=659515

But it should be removed